package-lock.json

```diff
@@ -12982,28 +12982,28 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": false,
+          "resolved": "",
```

Comment:
Do you have nvm set up to automatically switch to the right version? Because these changes look like they're caused by inconsistent npm versions. If that's because someone else used a different version than specified in `.nvmrc` then that's fine, but if it's because _you_ used a different version, this will cause such noise and potentially conflicts in future PRs :)

Reply:
Yes, I'm on the node version on the .nvmrc with this commit - but likely wasn't last time I updated package-lock.json. 😬